torch/_dynamo/variables/constant.py

```diff
@@ -113,6 +113,11 @@ def call_method(
             const_args = [a.as_python_constant() for a in args]
             const_kwargs = {k: v.as_python_constant() for k, v in kwargs.items()}
         except NotImplementedError:
+            if isinstance(self.value, list):
+                return variables.lists.ListVariable(
+                    self.value, mutable_local=variables.base.MutableLocal(), **options
+                ).call_method(tx, name, args, kwargs)
```

Comment:
Uh, so are you enforcing that lists are ListVariable or not lol

Reply:
Slash, why not put this with the rest of the self.value dispatch before the try-catch block

Reply:
We are except in the case of literals, for now.